app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+    before_filter :update_sanitized_params
+
+    def update_sanitized_params
+       devise_parameter_sanitizer.for(:sign_up) {|u| u.permit(:username, :email,   :password, :password_confirmation)}
```

Comment:
Line is too long. [118/80]<br>Unnecessary spacing detected.<br>Use 2 (not 3) spaces for indentation.<br>Space between { and | missing.<br>Space missing inside }.